crates/libs/core/src/strings/pcwstr.rs

```diff
@@ -26,14 +26,41 @@ impl PCWSTR {
         self.0.is_null()
     }
 
+    /// String length without the trailing 0
+    ///
+    /// # Safety
+    ///
+    /// The `PCWSTR`'s pointer needs to be valid for reads up until and including the next `\0`.
+    pub unsafe fn len(&self) -> usize {
+        #[cfg(target_os = "windows")]
```

Comment:
Other target cfg tests use the more concise `#[cfg(windows)]` (see hstring.rs) so it's probably best just to keep this consistent.